lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+class Registrant
+    attr_reader :name, :age, :permit, :license_data
+    def initialize(name, age, permit = nil)
+        @name = name
+        @age = age
+        @permit = permit unless permit.nil? do
```

Comment:
To set the default value of the permit to false, we'd want line 3 to read `def initialize(name, age, permit = false)` then line 6 would just be `@permit = permit` since we already set it to false on line 3.